prime-dotnet-webapi/Services/EmailRenderingService.cs

```diff
@@ -149,14 +149,22 @@ public async Task<Email> RenderSiteApprovedSigningAuthorityEmailAsync(string rec
 
         public async Task<Email> RenderSiteRegistrationSubmissionEmailAsync(LinkedEmailViewModel viewModel, CareSettingType careSettingCode)
         {
-            var recipientEmails = careSettingCode == CareSettingType.CommunityPharmacy
-                ? new[] { PrimeSupportEmail }
-                : new[] { MohEmail, PrimeSupportEmail };
+            if (careSettingCode == CareSettingType.CommunityPharmacy)
+            {
+                return new Email
+                (
+                    from: PrimeEmail,
+                    to: PrimeSupportEmail,
+                    subject: "PRIME Site Registration Submission",
+                    body: await _razorConverterService.RenderEmailTemplateToString(EmailTemplateType.SiteRegistrationSubmission, viewModel)
+                );
+            }
 
             return new Email
             (
                 from: PrimeEmail,
-                to: recipientEmails,
+                to: MohEmail,
+                cc: PrimeSupportEmail,
```

Comment:
This is now changing the behavior for community practice sites as well.

Reply:
Nathan and I had a discussion and it was that if we're sending to PrimeSupport then that can be in CC for all cases instead of in the To for both emails. Reason being that it's info for support but HIBC is the one that is taking an action